client/templates/minutes/minutesEditParticipants.html

```diff
@@ -15,6 +15,23 @@
             <!-- Expanded variant of participant list -->
             {{#if isParticipantsExpanded}}
                 <div class="padding-for-collabsable" id="id_participants">
+                    {{#if isModeratorOfParentSeries}}
+                        {{#if enoughParticipants}}
+                            <div class="checkbox" id="id_markAllNone">
+                                <label class = "checkbox-markAllNone">
+                                    <input type="checkbox" class="checkbox"
+                                           name="markAllNone" id="btnToggleMarkAllNone" checked="{{isChecked}}">
+                                    <span id="id_markAllNone" class="text-black">
```

Comment:
One more suggestion: to make even more clear that this is not a "normal" participant... could you maybe change the text color from back to "teal"? Its our primary application color - maybe it works here to distinguish this checkbox from the others?